site/docs/v1/tech/client-libraries/index.adoc

```diff
@@ -1,39 +1,70 @@
+Client libraries and SDks will help you quickly integrate your application with FusionAuth. All of our client libraries are open source and hosted on our https://github.com/FusionAuth[GitHub account]. You can fork and tweak them as well as look over the code to learn how the client libraries work.
 
+* <<SDKs vs Client Libraries>>
 * <<Languages>>
 * <<Usage Suggestions>>
 * <<Versioning>>
 
+== SDKs vs Client Libraries
+
+A **client library** is a thin wrapper over our FusionAuth APIs. It needs to cover all FusionAuth APIs. A client library is like a set of legos, to be put together by a developer who wants to extend or manipulate FusionAuth to meet their needs.
+
+You can use a client library to manage FusionAuth. For instance, if you wanted to rotate client secrets regularly, you could use a client library to do so.
+
+It can also be used to integrate with a custom application to offer login experiences that are different from those that are offered out of the box. If, in your app, you wanted to prompt someone for a username first, then do a lookup, then offer them a custom password field, then prompt them to enter their favorite color, use a client library to perform these complicated, custom operations.
+
+An **SDK** is an opinionated set of higher level constructs. These focus on a subset of functionality. These are like an assembled lego set.
+
+These let you quickly accomplish the common tasks and are often targeted at developers working on the front end: mobile/React/Vue/Angular/JavaScript developers. 
+
+FusionAuth SDKs have:
+
+* A button/function for logging in
+* A button/function for logging out
+* A button/function to register the user
+* A filter/some way to examine roles and limit information displayed to a given role or set of roles
+* A way to refresh a token without asking the user to reauthenticate.
+* Secure access and refresh token storage
+
+SDKs should require minimal customization to use.
+
+You can pair an SDK with a client library.
+
 == Languages
 
 If we are missing a language, open a https://github.com/FusionAuth/fusionauth-issues/issues[GitHub Issue] as a Feature Request if you don't see it already listed as an open feature.
 
-
 * link:/docs/v1/tech/client-libraries/dart[Dart]
 * link:/docs/v1/tech/client-libraries/go[Go]
 * link:/docs/v1/tech/client-libraries/java[Java]
-* link:/docs/v1/tech/client-libraries/javascript[JavaScript] (deprecated, prefer link:/docs/v1/tech/client-libraries/typescript[Typescript])
 * link:/docs/v1/tech/client-libraries/netcore[.NET Core]
-* link:/docs/v1/tech/client-libraries/node[Node] (deprecated, prefer link:/docs/v1/tech/client-libraries/typescript[Typescript])
-* link:/docs/v1/tech/client-libraries/openapi[OpenAPI] (**experimental**, please give us feedback)
+* link:/docs/v1/tech/client-libraries/openapi[OpenAPI]
 * link:/docs/v1/tech/client-libraries/php[PHP]
 * link:/docs/v1/tech/client-libraries/python[Python]
+* link:/docs/v1/tech/client-libraries/react-sdk[React SDK]
```

Comment:
Should we specify `Client Library` after the client libraries if we're going to put `SDK` after the SDKs?

Reply:
Good catch.